src/sidebar/components/login-form.js

```diff
@@ -3,13 +3,14 @@
 var angular = require('angular');
 
 // @ngInject
-function Controller($scope, $timeout, flash, session, formRespond, serviceUrl) {
+function Controller($scope, $timeout, analytics, flash, session, formRespond, serviceUrl) {
   var pendingTimeout = null;
 
   function success(data) {
     if (data.userid) {
       $scope.$emit('auth', null, data);
     }
+    analytics.track(analytics.events.LOGIN_SUCCESS);
```

Comment:
I see we're tracking login events in a component but logout events in a service. I can see why it was done that way but we should have a think about what layer analytics tracking ought to happen in.